packages/mui-lab/src/internal/pickers/wrappers/StaticWrapper.tsx

```diff
@@ -1,33 +1,76 @@
 import * as React from 'react';
 import { styled } from '@mui/material/styles';
+import {
+  unstable_composeClasses as composeClasses,
+  generateUtilityClass,
+  generateUtilityClasses,
+} from '@mui/core';
 import { DIALOG_WIDTH } from '../constants/dimensions';
 import { WrapperVariantContext, IsStaticVariantContext } from './WrapperVariantContext';
+import useThemeProps from '../../../../../mui-material/src/styles/useThemeProps';
+
+export interface StaticWrapperClasses {
+  /** Styles applied to the root element. */
+  root: string;
+}
+
+export type StaticWrapperClassKey = keyof StaticWrapperClasses;
+
+export function getStaticWrapperUtilityClass(slot: string): string {
+  return generateUtilityClass('MuiStaticWrapper', slot);
+}
+
+export const staticWrapperClasses: StaticWrapperClasses = generateUtilityClasses(
+  'MuiStaticWrapper',
```

Comment:
```suggestion
  'MuiPickerStaticWrapper',
```

I would propose this as a name. As you mentioned otherwise it's pretty generic and it is not clear that is related to the pickers components.

Reply:
The same would be used in the `useThemeProps` too.